Assets/Scripts/Crow/CrowManager.cs

```diff
@@ -94,6 +101,14 @@ private IEnumerator MoveCrow(GameObject crow)
     /// <returns>WaitForSeconds before hiding gameobject again</returns>
     private IEnumerator ShowAlert()
     {
+        //Initial reference to alert gameobject
+        if (!alertFound) 
```

Comment:
instead of having a variable for if the alert is found, you could just check if playerAlert is not null

Reply:
good point, have changed this to just check for null instead and removed the bool entirely